client/my-sites/email/titan-mail-quantity-selection/index.jsx

```diff
@@ -414,23 +356,20 @@ class TitanMailQuantitySelection extends React.Component {
 		return (
 			<>
 				<QueryProductsList />
+
 				{ selectedSite && <QuerySiteDomains siteId={ selectedSite.ID } /> }
+
 				<Main>
 					<DomainManagementHeader
 						onClick={ this.goToEmail }
 						selectedDomainName={ selectedDomainName }
 					>
-						{ translate( '%(productName)s: %(domainName)s', {
-							args: {
-								domainName: selectedDomainName,
-								productName: getTitanProductName(),
-							},
-							comment:
-								'%(productName)s is the name of the product, either "Email" or "Titan Mail"; %(domainName)s is the name of a domain',
-						} ) }
+						{ getTitanProductName() + ': ' + selectedDomainName }
```

Comment:
I'm not sure why we localized that string. We don't do it for G Suite by the way.